.github/workflows/proof_ci_resources/config.yaml

```diff
@@ -0,0 +1,7 @@
+cadical-tag: latest
```

Comment:
Does this file get used by anything other than the github actions? Could these config options be passed into the github action as parameters?

Reply:
The idea is that we'd like to be able to add new configuration parameters if we ever add more features to the proof CI Action.

We have a script to automatically update & preserve settings, but we'd rather have a separate file that contains only the settings, rather than writing over the `ci.yml` file that may be arranged in a particular way.

I'm not completely opposed to using parameters, but I think keeping them out of the ci.yml file is helpful when we want to update 14 repositories at once.

Reply:
My preference would be to split into two separate actions:

The first action would setup the runner with the necessary prerequisites (could be replaced by a container image in the future). This action would take arguments for the following:
* cadical-tag
* cbmc-version
* kissat-tag
* litani-version

With sane defaults in the definition in the CI-CD-Github-Actions repository so most spoke repositories don't need to set anything specific.

The second action would run set of CI proofs given the following arguments:
* proofs-dir (with a reasonable default?)
* proofs (list of proofs to run or run all by default).
* script-name (if needed... I think it's probably run-cbmc-proofs.py in all of the spoke repositories right?)

Reply:
The second action could also probably just be a normal shell-script step. Maybe thats easier?